### DISCUSSION TITLE
darxiのx3とx4

### DISCUSSION THREAD

**Discussion by niftg** [ponjbogri/jvs_ja#1] [General]
> x1 は x2 （対象本体）・ x3 （対象箇所）を x4 で打つ／叩く

現版では☝️とありますが、英語版との一貫性においては「x3でx4（対象箇所）」とすべきかと見えます（つまりx3とx4の説明が逆転してる）。
noteでは“x3は「手」「指」「足」などの肉体部分も。”とあり、箇所よりは道具寄りの話であるので、意図的に逆にしたPSで再定義したと見るよりは単に翻訳時のミスと見る方が自然な気がします。
なのですぐにJVSのデータを書き直すことは可能なのですが、念のため一旦ここに書き残しておきます。mu'o

(書きそびれてましたが、報告してくださったvevさんにki'esai.ioです)

**Comment by niftg** [comment]
ちなみにpencuの現版も同様の一貫性の問題があるみたいですね…

- > x1 (agent) touches x2 with x3 [a locus on x1 or an instrument] at x4 [a locus on x2].
- > x1 は x2 （対象本体）・ x3 （対象箇所）を x4 で触る

**Comment by guskant** [comment]
同意です。jvsの日本語訳を修正する必要がありますね。

**Reply by niftg** [reply]
ひとまづ項の入替のみしてみました

darxi:
> $x_1$ は $x_2$ （対象本体）・ $x_3$ （対象箇所）を $x_4$ で打つ／叩く

↓

> $x_1$ は $x_2$ （対象本体）を $x_3$ で $x_4$（対象箇所）に接して打つ／叩く

pencu:
> $x_1$ は $x_2$ （対象本体）・ $x_3$ （対象箇所）を $x_4$ で触る

↓

> $x_1$ は $x_2$ （対象本体）を $x_3$ で $x_4$（対象箇所）に接して触る

**Reply by niftg** [reply]
（この位置にx4を嵌め込んで自然な日本語にするのが案外難しくて、さしあたり`接する`でお茶を濁したのですが、他に何か良い表現があれば採り入れたいです）

**Reply by guskant** [reply]
もともと $x_3$ の説明を notes 欄でしている形式を再利用し、 definition をシンプルにして、 notes で $x_3$, $x_4$ を説明すると読みやすいかもしれません。
例えば以下のような形です。防具や衣服は英語定義に明示されていないので賛否あるかもしれませんが、英語の hit や touch の一般的な使われ方から類推しました。

definition
$x_1$ は $x_2$ を $x_3$ が $x_4$ にぶつかる形で打つ／叩く
notes
$x_3$ は $x_1$ の一部、 $x_4$ は $x_2$ の一部であり、武器や防具などを含む。
 
definition
$x_1$ は $x_2$ を $x_3$ が $x_4$ に接する形で触る
notes
$x_3$ は $x_1$ の一部、 $x_4$ は $x_2$ の一部であり、道具や衣服などを含む。

**Reply by niftg** [reply]
なるほど綺麗ですね！それで行きたい気はしたのですが、無駄が削ぎ落とされたあまり初見で意味を掴み損ねる恐れも覚えたので、darxiについてはx3に`(道具など)`の形で行内補註を復活させたくも感じます。(x2の件はやはりnotesのが向いてるかもしれません)

また、賛否の予想される類推事項とのことなので、「を含む」を少し和らげて「も挙げられる」「も考えられる」などとして、断定度を更に下げてみたく感じました。

以上を考慮して調整した文言が以下になります：

definition
$x_1$ は $x_2$ を、 $x_3$ (道具など) が $x_4$ にぶつかる形で打つ／叩く
notes
$x_3$ は $x_1$ の一部、 $x_4$ は $x_2$ の一部であり、それぞれ武器や防具なども挙げられる。

definition
$x_1$ は $x_2$ を、 $x_3$ が $x_4$ に接する形で触る
notes
$x_3$ は $x_1$ の一部、 $x_4$ は $x_2$ の一部であり、それぞれ道具や衣服なども挙げられる。

**Reply by guskant** [reply]
ほぼ賛成します。1つだけ、自分で書いていたのですが definition 内の「触る」の助詞は「を」より「に」が適切だったかもしれません。
$x_1$ は $x_2$ に、 $x_3$ が $x_4$ に接する形で触る

**Reply by niftg** [reply]
.u'o ki'e .io
そちらのpencuの定義にして再調整した上で昨日再登録しました.uo。（definition欄以外では$記法が非推奨の様なので渋々`$x_1$`→`x1`とはしましたが…）

(追記: 一応notesにも$記法は残せてHTMLには反映される様ですが、印刷版.a'acu'iの出力に影響があるみたいですね)
